src/pages/RepoPage/CoverageTab/subroute/Fileviewer/Fileviewer.jsx

```diff
@@ -34,6 +29,7 @@ function FileView() {
         sticky
         stickyPadding={215}
         showFlagsSelect={showFlagSelector}
+        showComponentsSelect
```

Comment:
Just curious, my frontend is lacking. Why doesn't this also accept a var like `{showComponentsSelector}`?

Reply:
Oh shoot you're right, this needs to be fixed @rohitvinnakota-codecov I totally missed this. Unless you're passing the value directly as @thomasrockhu-codecov is suggesting it will always be true, which is not always the case.